<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ingreso</class>
 <widget class="QDialog" name="Ingreso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ingreso</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>430</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="IngEditar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Agente">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>Seleccionar</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="IngCant">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RepBorrar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>53</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="RepTabla">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>641</width>
     <height>91</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Agente</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>FechaIng</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>FechaRep</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Operario</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Presupuesto</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="IngGuardar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_49">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>140</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Codigo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="IngBorrar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lID_33">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>76</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Fecha Ingreso</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_50">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cantidad</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="ID_Rep">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDate|Qt::ImhSensitiveData</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_51">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>190</y>
     <width>190</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Diagnostico Cliente</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="IngresoTabla">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>701</width>
     <height>151</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Equpo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>SN</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cant</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Factura</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Diagnostico Cliente</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>RepID</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="RepEditar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>53</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="IngCom">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RepGuardar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>53</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FIngreso">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDate|Qt::ImhSensitiveData</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>37</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Agente</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="IngEquipo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>Seleccionar</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lID_34">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>33</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>RepID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="IngSN">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="IngFac">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_52">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>SN</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_53">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>190</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Factura/Remito</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Ingreso</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Ingreso</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
